setup/raspberry-pi/ansible-playbook.yaml

```diff
@@ -96,3 +100,27 @@
         sudo make install && \
         rm -rf git-2.5.0*
       tags: git
+
+    - name: git | Download, compile and install ccache 3.2
+      shell: rm -rf ccache-3.2.3 && \
+        curl -sL http://samba.org/ftp/ccache/ccache-3.2.3.tar.xz | tar -Jxv && \
+        cd ccache-3.2.3/ && \
+        ./configure --prefix=/usr --sysconfdir=/etc -q && \
+        make && \
+        sudo apt-get remove ccache -y && \
+        sudo make install && \
+        sudo mkdir -p /usr/lib/ccache && \
+        sudo ln -sf $(which ccache) /usr/lib/ccache/gcc && \
+        sudo ln -sf $(which ccache) /usr/lib/ccache/cc && \
+        sudo ln -sf $(which ccache) /usr/lib/ccache/g++ && \
+        sudo ln -sf $(which ccache) /usr/lib/ccache/c++ && \
+        rm -rf ccache-3.2.3
+      tags: ccache
```

Comment:
yeah, should be, I'm trying that out now on the armv8 machines to see how it goes then I'll replace this if it's all good there